src/venueless-help-page.html

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Help Page
+description: Find Help
+---
+
+<div class="hero">
+  <div class="wrapper">
+    <header class="flex flex-wrap items-center gap-8 lg:gap-16">
+      <img
+        src="/assets/img/theme/hero-img.webp"
+        alt="welcome"
+        class="hidden w-full max-w-80 md:block" />
+      <div class="flex-1 max-w-screen-lg">
+        <h1 class="mb-8 pageheading">{{ title }}</h1>
+        <p class="subheading">
+          Need Help? This is the place! Or, the go to <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack</a> Page.
+        </p>
+      </div>
+    </header>
+  </div>
+</div>
+<!-- <div class="bg-gray-100 block-container">
+  <div class="wrapper">
+    <div class="flex items-center justify-center min-h-screen">
+      <div class="gap-8 list-disc lg:gap-16 md:columns-2 prose lg:prose-lg *:break-inside-avoid">
+          <p>General Venuless Questions?</p>
+          <p>Come Visit Our Help Desk in <a href="https://join.slack.com/t/djangoconus2024/shared_invite/zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ">Slack </a></p>
```

Comment:
We can't have the link on our repo because then people who don't have tickets can join

Reply:
We are porting this over to an iframe in Venueless, they should already have tickets if they are seeing this page.

Reply:
@blulady I've deactivated that link (zt-2qh4h5fgg-m4HbkbtNWBSm2Kq3nG~7yQ) You'll need to generate a new one if you want to invite someone to the slack.